### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stenioaraujo**

### CODE REVIEW

**Comment by lets00** [comment]
Ok!

File: README.md [hunk 1]
```diff
@@ -6,6 +6,6 @@ They are intended to be used as is. This makes life easy.
 
 ## List of projects
 
-### **road**
```
Comment (stenioaraujo) [inline README.md:9]: I removed the bold part, it is not needed.

Comment (lets00) [inline README.md:9]: Mudar aqui

Comment (lets00) [reply README.md:9]: Oks

**Pull request closed without merging**